docs/limitations.md

```diff
@@ -33,6 +33,8 @@
     * [runtime commands](#runtime-commands)
         * [`init` command](#init-command)
         * [`spec` command](#spec-command)
+* [Other notes](#other-notes)
+    * [Host `rdmsr` warnings](#host-rdmsr-warnings)
 
 As Intel® Clear Containers utilises Virtual Machines (VM) to enhance
```

Comment:
Sub "utilises" with "utilizes"

Reply:
@klynnrif - that change is not really part of this PR - it is unrelated (already existed). No doubt it emanated from the English'ness of @jodh-intel and myself :-)
I'll add that as an add-on fix to this PR - but we should be very aware of reviewing and feeding back information generally only pertinent to the PR in question - if other major things pop out to you then it is probably best to open a new Issue to request them to be fixed.

Reply:
Yes, the "s" variant is much more common I think over here.

At the risk of starting a language war... imho if the word is in the [OED](https://en.wikipedia.org/wiki/Oxford_English_Dictionary) (and both spellings are listed for this word), we should not try to Americanise it (or should that be Americanize? :smile:) and consider accepting both spellings. However, there is something to be said for consistency. Looking towards kata, I think we can avoid a lot of this confusion going forward if we define our approach somewhere (probably https://github.com/kata-containers/documentation (either wiki or in-tree)).

See the related issue I raised way back on this: https://github.com/01org/cc-oci-runtime/issues/788.

Reply:
@grahamwhaley @jodh-intel Thanks for the explanation :) I figured that was the case and did question myself before including it. I agree with @jodh-intel on no war and consistency :) Thanks!